pkgs/test_core/README.md

```diff
@@ -1,5 +1,5 @@
-## test_core
+A minimal package for writing and running tests also with extensions for
```

Comment:
nit: this reads weird now, maybe `s/with/as well as`?

Reply:
ack